### PR TITLE
UK TL: More new options

### PR DESCRIPTION
**Pull request by tsunamistate**
@SourceCodeDeleted please take a look

### CODE REVIEW

**Comment by AJenbo** [comment]
68%, it's getting close to the magic 75% required to be listed :)

@SourceCodeDeleted do you have time to review the update?

File: Translations/uk.po [hunk 1]
```diff
@@ -1200,145 +1208,142 @@ msgstr "Допомога недоступна"
...
+#: Source/diablo.cpp:1406
 msgid "Hotkey for skill or spell."
 msgstr "Гаряча клавіша для чар або таланту."
 
-#: Source/diablo.cpp:1416
-msgid "Quick Message {}"
-msgstr "Швидке Послання {}"
+#: Source/diablo.cpp:1423
+msgid "Speedbook"
+msgstr "Швидка Книга Магії"
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:1229]: The rest of the translateion uses
Чар
Can we continue to use this word?

File: Translations/uk.po [hunk 2]
```diff
@@ -1200,145 +1208,142 @@ msgstr "Допомога недоступна"
...
+#: Source/diablo.cpp:1480
 msgid "Open Spellbook."
 msgstr "Відкрити Книгу Чар."
 
-#: Source/diablo.cpp:1484
-msgid "Game info"
-msgstr "Інформація про гру"
+#: Source/diablo.cpp:1487
+msgid "Quick Message {}"
+msgstr "Швидке Послання {}"
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:1297]: Do you think повідомлення would be better?

File: Translations/uk.po [hunk 3]
```diff
@@ -1200,145 +1208,142 @@ msgstr "Допомога недоступна"
...
 
-#: Source/diablo.cpp:1485
-msgid "Displays game infos."
-msgstr "Показує інформацію про гру."
+#: Source/diablo.cpp:1488
+msgid "Use Quick Message in chat."
+msgstr "Відправити Швидке Послання в чат."
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:1301]: повідомлення perhaps?

**Comment by SourceCodeDeleted** [comment]
Ok most of this looks good. @tsunamistate I made a small suggestion, how do you think?>

File: Translations/uk.po (hunk 2 above)
Comment (tsunamistate) [reply Translations/uk.po:1297]: Too long

File: Translations/uk.po (hunk 1 above)
Comment (tsunamistate) [reply Translations/uk.po:1229]: Fixed

File: Translations/uk.po (hunk 3 above)
Comment (tsunamistate) [reply Translations/uk.po:1301]: I thought it was too long, but here's the screenshot with your suggestion
image

Looks OK, so I have changed it to your suggestion

File: Translations/uk.po (hunk 2 above)
Comment (tsunamistate) [reply Translations/uk.po:1297]: Done

**Comment by tsunamistate** [comment]
Fixed inconsistencies pointed out. Turns out I left one TL incomplete, so I fixed that too.
@SourceCodeDeleted as for your suggestion, I thought it would be too long in UI, but play testing proved me wrong, so I changed it.
@AJenbo fixed and rebased, good to merge

**Pull request merged**